tests/functional/firefox/test_win10_welcome.py

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.win10_welcome import Win10WelcomePage
+
+
+@pytest.mark.skip_if_firefox
```

Comment:
It's not clear to me why we're skipping if Firefox.. perhaps an additional comment would help? How would the test fail if we did run it in Firefox?